src/data/set/basic.lean

```diff
@@ -1621,6 +1621,13 @@ begin
     convert congr_arg σ h; exact (σ.apply_symm_apply _).symm }
 end
 
+/-- The subtype of subsets of `t` is equivalent to `set ↥t`. -/
```

Comment:
This _maybe_ would fit better in logic/equiv/set; possibly even states in the reverse direction since the type currently on the LHS is less nested and therefore simpler.

Reply:
The type on the LHS is certainly harder to work with, though, so I think this is the correct direction.